runtime/doc/if_lua.txt

```diff
@@ -13,14 +13,17 @@ Lua Interface to Nvim					*lua* *Lua*
 
 Neovim lua interface automatically adjusts `package.path` and `package.cpath` 
 according to effective &runtimepath value.  Adjustment happens after each time 
-'runtimepath' is changed, `package.path` and `package.cpath` are adjusted by 
-prepending directories from 'runtimepath' each suffixed by `/lua` and 
-`?`-containing suffixes from `package.path` and `package.cpath`. I.e. when 
-'runtimepath' option contains `/foo` and `package.path` contains only 
-`./?.lua;./a?d/j/g.nlua;/bar/?.lua` the resulting `package.path` after 
+'runtimepath' is changed. `package.path` is adjusted by simply appending 
+`/lua/?.lua` and `/lua/?/init.lua` to each directory from 'runtimepath' (`/` 
+is actually a first character from `package.config`).
+
+`package.cpath` is adjusted by prepending directories from 'runtimepath' each 
+suffixed by `/lua` and `?`-containing suffixes from existing `package.cpath`. 
+I.e. when 'runtimepath' option contains `/foo` and `package.cpath` contains 
+only `./?.so;./a?d/j/g.elf;/bar/?.so` the resulting `package.cpath` after 
```

Comment:
I still find this wording hard to parse: `is adjusted by prepending ... each suffixed by ... -containing suffixing from`. This is work for a native English speaker, but I'll give it a shot:

```
Similarly to `package.path`, modified directories from `runtimepath` are also added to `package.cpath`. In this case, instead of appending `/lua/?.lua` and `/lua/?/init.lua` to each runtimepath, all unique `?`-containing suffixes of the existing `package.cpath` are used. An example explains it better: 

1. 'runtimepath' option contains `/foo`
2. `package.cpath` is `./?.so;./a?d/j/g.elf;/bar/?.so`, which means the unique `?`-suffixes are `/?.so` (from `./?.so` and `/bar/?.so`) and `/a?d/j/g.elf` (from `./a?d/j/g.elf`). 
3. Thus the adjusted `package.cpath` looks like:  

  /foo/lua/?.so;/foo/lua/a?d/j/g.elf;./?.so;./a?d/j/g.elf;/bar/?.so
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
             runtimepath derived        ^^^^^^^^^^^^^^^^^^^^^^^^^
                                                         original package.cpath
```

I realize this is also in poor shape. I just believe that though the Lua pattern is easy to grok, any text describing it is not. Which is why emphasizing the example is worth a War and Peace IMHO. Any suggestions, @justinmk?